packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -130,3 +130,92 @@ export const glowClass = style([
     zIndex: 0,
   },
 ]);
+
+export const selectContainerClass = style([
+  sprinkles({
+    alignItems: 'stretch',
+    backgroundColor: '$gray90',
+    borderRadius: '$sm',
+    display: 'flex',
+    flexGrow: 1,
+    lineHeight: '$lg',
+    overflow: 'hidden',
+    paddingLeft: '$4',
+    paddingRight: '$2',
+    position: 'relative',
+  }),
+  {
+    border: `1px solid ${vars.colors.$gray40}`,
+    selectors: {
+      '&:active': {
+        color: vars.colors.$gray40,
+      },
+    },
+  },
+]);
+
+export const selectContainerClassDisabled = style([
+  sprinkles({
+    backgroundColor: {
+      lightMode: '$gray20',
+    },
+    color: {
+      lightMode: '$foreground',
+    },
+  }),
+]);
+
+export const selectIconClass = style([
+  sprinkles({
+    alignItems: 'center',
+    display: 'flex',
+    color: '$gray40',
+  }),
+  {
+    selectors: {
+      '&:active': {
+        color: vars.colors.$gray40,
+      },
+    },
+  },
+]);
+
+export const selectClass = style([
+  sprinkles({
+    background: 'none',
+    border: 'none',
+    color: '$gray40',
+    flexGrow: 1,
+    outline: 'none',
```

Comment:
I think we do want an outline for when you tab to the element? Currently the rest of the navHeader can be navigated by tabbing and all the items are visually highlighted when you do except this one.

Not sure though, @eileenmguo ?

Reply:
Yeah, this will have an outline, but it will match the style of the input and select component that @isaozler is designing so that update will be included in that ticket